webapp/components/test/test-search.html

```diff
@@ -361,6 +361,21 @@
           }],
       });
     });
+
+    test('multi-root', () => {
+      assertQueryParse('none(status:missing) count>0(status:!pass)', {
```

Comment:
Can we also support explicit and at this level? Follow-up is fine.

Reply:
Already had, just wasn't documented.